src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -417,40 +408,33 @@ const renderEstimate = (estimatedPrice, user, mediator, size) => {
               intent: Intent.seeEstimateAuctionRecords,
             })
         }}
+        textDecoration="underline"
       >
         <Sans size="2">Sign up to see estimate</Sans>
-      </Link>
+      </Clickable>
     )
   }
+
+  return <Flex justifyContent="flex-start">{body}</Flex>
 }
 
 const renderRealizedPrice = (
   estimatedPrice,
   user,
   mediator,
-  size,
   filtersAtDefault
 ) => {
-  const justifyContent = size === "xs" ? "flex-start" : "flex-end"
   // Show prices if user is logged in. Otherwise, show prices only when filters at default.
+  let body: JSX.Element
   if (user || filtersAtDefault) {
-    return (
-      <Flex justifyContent={justifyContent}>
-        {estimatedPrice && (
-          <>
-            <Sans size="2">{estimatedPrice}</Sans>
-          </>
-        )}
-        {!estimatedPrice && (
-          <>
-            <Sans size="2">Price not available</Sans>
-          </>
-        )}
-      </Flex>
-    )
+    if (estimatedPrice) {
```

Comment:
**nitpick [if-minor]:** wrong name for local

I think there was a copy/paste mistake here I was hoping you might clean up for us. In this component, I'd expect for this var to be called `realizedPrice`. Should be an easy find/replace but no biggie if you want to skip it.

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
good catch, I'll fix this.

Reply:
fixed in the latest commit